<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>603</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>603</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AppIGen</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fira Sans</family>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="line_edit_image_path">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_edit_project_path">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>280</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_image">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_select_project">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apply_changes">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Apply Changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>145</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Select Image for App</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>62</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Select Project Folder</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>212</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border : 1px solid grey;
	border-radius: 10px;

}
</string>
    </property>
    <property name="title">
     <string>Selected Image</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_image_display">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>26</y>
       <width>200</width>
       <height>170</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Fira Sans</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_img_info">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Status :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>325</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Waiting for app image...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>121</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Image Name :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Image Size :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Image Type :&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Image Resolution : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Sans</family>
     </font>
    </property>
    <property name="text">
     <string>Reset </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
